src/app/components/Item/ItemTable.jsx

```diff
@@ -56,13 +56,12 @@ const ItemTable = ({ items, holdings, bibId, id, searchKeywords, page }) => {
           </table>
           {
             page === 'SearchResults' &&
-            <RequestButtons
+            <StatusLinks
               item={group[0]}
               bibId={bibId}
               searchKeywords={searchKeywords}
               appConfig={appConfig}
```

Comment:
can we just import appconfig within the statuslink components instead of passing it around?

Reply:
doing it this way makes it easier to test different scenarios, otherwise I would agree.